images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -21,8 +22,10 @@ function Set-JavaPath {
     Write-Host "Set 'JAVA_HOME_${Version}_X64' environmental variable as $javaPath"
     setx JAVA_HOME_${Version}_X64 $javaPath /M
 
-    if ($Default)
+    if ($DefaultVersion)
     {
+        # Here we remove any jdk versions installed prior to the main java installation (say, by Choco)
+        # then we rebuild the %PATH% variable value back to the normal one
```

Comment:
```suggestion
        # Clean up any other Java folders from PATH to make sure that they won't conflict to each other
```

Please try to avoid using `here` and `we` in comments.

Reply:
Agreed